CHANGES.rst

```diff
@@ -34,6 +34,8 @@ astropy.io.misc
 astropy.io.fits
 ^^^^^^^^^^^^^^^
 
+- Fix a bug in io.fits with writting fortan-ordered arrays in python3 when np.nditer was available [#8282]
```

Comment:
Nitpicks to the change log text. :grimacing: Also, please move this entry to the section indicated in milestone.
```suggestion
- Fix a bug in ``io.fits`` with writting Fortan-ordered arrays in Python 3 when
  ``np.nditer`` was available. [#8282]
```